From: Kerin Millar <kerframil@gmail.com>

Disable CONFIG_COMPAT_VDSO entirely. It is inappropriate for any Gentoo 
user to activate this option. Moreover, it prevents users from selecting 
a number of important PaX options - notably PAX_PAGEEXEC and 
PAX_SEGMEXEC. Under these circumstances, it is impossible for the user 
to enforce non-executable pages. Unfortunately, this is far from obvious 
to first-time users. Closes bug 210138. 

--- a/arch/i386/Kconfig	2008-02-14 17:46:47.000000000 +0000
+++ b/arch/i386/Kconfig	2008-02-14 17:57:03.000000000 +0000
@@ -915,16 +915,8 @@
 	  /sys/devices/system/cpu.
 
 config COMPAT_VDSO
-	bool "Compat VDSO support"
+	bool
 	default n
-	help
-	  Map the VDSO to the predictable old-style address too.
-	---help---
-	  Say N here if you are running a sufficiently recent glibc
-	  version (2.3.3 or later), to remove the high-mapped
-	  VDSO mapping and to exclusively use the randomized VDSO.
-
-	  If unsure, say Y.
 
 endmenu
 
